Library/Homebrew/os/linux/elf.rb

```diff
@@ -126,8 +127,20 @@ def initialize(path)
 
     private
 
+    def default_on_exception(val, exception)
+      yield
+    rescue exception
```

Comment:
Catching all exceptions without checking the type makes me nervous. Can we be more specific on the type of exception that we catch?

Reply:
@sjackman yes, we don't catch all exception, we only catch the exception passed to `default_on_exception` which happens to be `PatchELF::PatchError` in this case.

Reply:
Ah, I see. Thanks for the explanation.